README.md

```diff
@@ -124,7 +124,7 @@ If your project does not use a license, you may opt out of specifying one:
 bintrayOmitLicense := true
 ```
 ##### public (default)
-If your project uses a license, Bintray supports those listed [here](https://bintray.com/docs/api/#_footnote_1). If you are new to software licenses you may
+If your project uses a license, Bintray supports those listed on the bottom of the [Bintray API documentation](https://bintray.com/docs/api/). If you are new to software licenses you may
```

Comment:
We can use https://bintray.com/docs/api/#_licenses

Reply:
or better: https://bintray.com/docs/api/#_get_oss_licenses ?

Reply:
Sure!

Reply:
I've changed it.